rust/processor/migrations/2024-05-04-025823_current_unified_fungible_asset_balance/up.sql

```diff
@@ -0,0 +1,21 @@
+-- current fungible asset balances
+CREATE TABLE IF NOT EXISTS current_unified_fungible_asset_balances (
+    storage_id VARCHAR(66) PRIMARY KEY NOT NULL,
+    owner_address VARCHAR(66) NOT NULL,
+    asset_type VARCHAR(66) NOT NULL,
+    coin_type VARCHAR(1000),
+    is_primary BOOLEAN,
+    is_frozen BOOLEAN NOT NULL,
+    amount_v1 NUMERIC,
+    amount_v2 NUMERIC,
+    amount NUMERIC GENERATED ALWAYS AS (COALESCE(amount_v1, 0) + COALESCE(amount_v2, 0)) STORED,
+    last_transaction_version_v1 BIGINT,
+    last_transaction_version_v2 BIGINT,
```

Comment:
I forget, why do we need `last_transaction_version_v1` and `last_transaction_version_v2`?

Reply:
to know the latest version that updates each part. You need this to update based on inserted_entry_time >= latest.